spec/services/journal_row_updater_spec.rb

```diff
@@ -17,4 +17,35 @@
     end
   end
 
+  describe "when the order detail is on a failed journal" do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:facility) { FactoryGirl.create(:setup_facility) }
+    let(:account) { FactoryGirl.create(:setup_account, owner: user) }
+    # Original costs: actual: 20, subsidy: 10
```

Comment:
I wonder if instead of adding this comment, we should explicitly setup these values in the test?  Unless it's a lot of hassle.